### DISCUSSION TITLE
Missing $ Inserted

### DISCUSSION THREAD

**Discussion by rmfmarinho** [joaomlourenco/novathesis#484] [Q&A] [answered]
I've had this error for some time in my document, and I can't understand where it comes from:

Version 2023-02-11 [6.10.18]Win11
Windows
Overleaf
FCT NOVA Msc

![erro](https://github.com/joaomlourenco/novathesis/assets/125412616/dd9c5203-b620-4339-9144-355875e11613)

It redirects me to line 235 of template.tex, which I haven't changed.

![erro2](https://github.com/joaomlourenco/novathesis/assets/125412616/99b4067d-1792-409f-8b95-fa1db7c67f90)

**Comment by joaomlourenco** [comment]
In the red bar on the top: `./Chapters/chapter3.texNT, 235`
Did you check that line?

**Reply by rmfmarinho** [reply]
Yes. It takes me to a tikz diagram I wrote. I deleted it and I still have the same error. In fact, I deleted everything in chapter 3 and I still have the same errors.

**Reply by rmfmarinho** [reply]
![erro3](https://github.com/joaomlourenco/novathesis/assets/125412616/3d0373df-908d-4008-a626-c84738aacdf5)

I saw these errors related to under scores in novathesis.cls. Could this be the problem?

**Reply by joaomlourenco** [reply, marked as answer]
The (original) template compiles and has no errors.
You have an error _somewhere_ in your document that is messing up with the LaTeX compiler.
In `Config/4_files.tex`, comment the adding of the chapters and recompile, then uncomment one-by-one and see which chapter is messing up with LaTeX. But I'm guessing it s `Chapter 3`.

**Reply by rmfmarinho** [reply]
Thank you. Turns out it was an image in chapter 2. Wasn't showing any error, so I had to search line by line 😆.
